ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -108,6 +109,48 @@ def monitored(self):
                     if self.getMonitored(datasource):
                         return True
 
+        # 3 check the service class
+        if hasattr(self, 'serviceclass') and 'serviceclass' in self.getRelationshipNames():
+            sc = self.serviceclass()
+            if sc:
+                org = sc.serviceorganizer()
+                # check the service class monitored start modes
+                if self.startMode in sc.monitoredStartModes:
+                    # check the zMonitor organizer property
+                    if org and hasattr(org, 'zMontitor'):
```

Comment:
I believe so, it's from ZenModel/ServiceOrganizer.py and the Service Class page has:

Enable Monitoring? (zMonitor)